tools/ui-components/src/button/button.tsx

```diff
@@ -189,6 +189,7 @@ export const HeadlessButton = React.forwardRef<
           download={download}
           target={target}
           ref={ref as React.Ref<HTMLAnchorElement>}
+          onClick={onClick}
```

Comment:
Passing an onClick to an anchor element seems like an accessibility anti-pattern.

Reply:
> The fix here is to pass `onClick` down to the base `<a>`. I also added a unit test to cover this behavior.

I'm not sure that's the fix. Instead, why don't we make the button allow `href` or `onclick`, but not both? In that case we'd have to refactor `ProjectsFor` slightly (it would need to pick which of the two to render based on `isCert`), but that seems fine.

How does that sound?

Reply:
Serves me right for picking one line out of context. Sorry about that!

So, as long as we're keeping the accessibility we have, I'm fine with this going in as is.

As for where we go from here, I don't have a strong opinion about what's better: 1) a `Button` that accepts both `href` and `onclick`, but not both, or 2) a `ButtonLink` that does not accept `onclick` and a `Button` that does. But, yes, if we do go with 1) I agree that the current interface isn't good.

Reply:
I know that feeling. The api migration faced the exact same issue and it's been a slog.

Aside from giving this a review, is there anything else I can help with?